common/src/thrift/raw_table_master.thrift

```diff
@@ -19,7 +19,8 @@ service RawTableMasterService {
   /*
    * Creates a raw table.
    */
-  i64 createRawTable(1: string path, 2: i32 columns, 3: binary metadata)
+  i64 createRawTable(1: string path, 2: i32 columns, 3: binary metadata, 
+      4: common.RpcOptions rpcOptions)
```

Comment:
Moved to first argument.

Right now I just want the build to stop failing so I'm changing only this method. I'll follow up with another PR doing this for all RPCs.